pkg/provider/azure_vmss.go

```diff
@@ -1825,3 +1853,17 @@ func (ss *ScaleSet) GetNodeVMSetName(node *v1.Node) (string, error) {
 	klog.V(4).Infof("ss.GetNodeVMSetName: found vmss name %s from node name %s", vmssName, node.Name)
 	return vmssName, nil
 }
+
+// VMSSBatchSize returns the batch size for VMSS operations.
+func (ss *ScaleSet) VMSSBatchSize(vmssName string) (int, error) {
+	batchSize := 1
```

Comment:
should we use the same default value `0` here as the cloud config file?

Reply:
sure and fixed